changelog.d/15889.bugfix

```diff
@@ -0,0 +1 @@
+Remove outlier check for getting membership events through the `/_matrix/client/r0/rooms/{roomId}/event/{eventId}` endpoint.
```

Comment:
This PR probably needs more eyes on whether we want to do this so I've put it back in the review queue